### PR TITLE
[IMP] sale_order_line_description: res_config => new api

### PR DESCRIPTION
**Pull request by Daniel-CA**
res_config to new api

### CODE REVIEW

File: sale_order_line_description/models/res_config.py [hunk 1]
```diff
@@ -19,19 +19,16 @@
...
-            """Allow using only the product sale description
-            on the sales order lines""",
-            implied_group="sale_order_line_description."
-            "group_use_product_description_per_so_line",
-            help="""Allows you to use only product sale description on the
-            sales order line."""
-        ),
-    }
+    group_use_product_description_per_so_line = fields.Boolean(
+        """Allow using only the product sale description on the sales
```
Comment (pedrobaeza) [inline sale_order_line_description/models/res_config.py]: Don't put triple quotas here, and change translations template according it.

**Comment by Daniel-CA** [comment]
Done!

File: sale_order_line_description/i18n/sale_order_line_description.pot [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#. module: sale_order_line_description
+#: field:sale.config.settings,group_use_product_description_per_so_line:0
+msgid "Allow using only the product sale description on the sales\n"
+"        order lines"
+msgstr ""
+
+#. module: sale_order_line_description
+#: help:sale.config.settings,group_use_product_description_per_so_line:0
+msgid "Allows you to use only product sale description on the\n"
+"        sales order line."
```
Comment (pedrobaeza) [inline sale_order_line_description/i18n/sale_order_line_description.pot]: This is the old translation. The new mustn't contain spaces.

File: sale_order_line_description/i18n/sale_order_line_description.pot [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: sale_order_line_description
+#: field:sale.config.settings,group_use_product_description_per_so_line:0
+msgid "Allow using only the product sale description on the sales\n"
+"        order lines"
```
Comment (pedrobaeza) [inline sale_order_line_description/i18n/sale_order_line_description.pot]: The same

**Comment by Daniel-CA** [comment]
Done, I forgot the update

**Comment by pedrobaeza** [comment]
:+1:

**Comment by eLBati** [comment]
thanks :+1:

**Comment by tafaRU** [comment]
:+1:

**Pull request merged**